playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/digitalgoods/ListPurchasesCall.java

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.androidbrowserhelper.playbilling.digitalgoods;
+
+import android.os.Bundle;
+import android.os.Parcelable;
+
+import com.android.billingclient.api.BillingClient;
+import com.android.billingclient.api.BillingResult;
+import com.android.billingclient.api.Purchase;
+import com.google.androidbrowserhelper.playbilling.provider.BillingWrapper;
+
+import java.util.List;
+
+import androidx.annotation.Nullable;
+
+/**
+ * A class for parsing Digital Goods API calls from the browser and converting them into a format
+ * suitable for calling the Play Billing library.
+ */
+public class ListPurchasesCall {
+    public static final String COMMAND_NAME = "listPurchases";
+
+    static final String RESPONSE_COMMAND = "listPurchases.response";
+    static final String KEY_PURCHASES_LIST = "listPurchases.purchasesList";
+    static final String KEY_RESPONSE_CODE = "listPurchases.responseCode";
+
+    private final DigitalGoodsCallback mCallback;
+
+    private ListPurchasesCall(DigitalGoodsCallback callback) {
+        mCallback = callback;
+    }
+
+    /** Creates this class from a {@link Bundle}, returns {@code null} if the Bundle is invalid. */
+    @Nullable
+    public static ListPurchasesCall create(@Nullable Bundle args,
```

Comment:
The args param is unused. Will it be used in future? Otherwise perhaps delete it?

Reply:
Good catch - I was just copying the format of `GetDetailsCall`, but it is unnecessary.